Exception on /user/charge [POST]
Traceback (most recent call last):
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/kapil.rajput/Downloads/prototype/flaskSaaS-master/app/views/user.py", line 176, in charge
    customer = stripe.Customer.create(email=current_user.email, source=request.form['stripeToken'])
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/stripe/api_resources/abstract/createable_api_resource.py", line 22, in create
    response, api_key = requestor.request("post", url, params, headers)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/stripe/api_requestor.py", line 121, in request
    resp = self.interpret_response(rbody, rcode, rheaders)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/stripe/api_requestor.py", line 372, in interpret_response
    self.handle_error_response(rbody, rcode, resp.data, rheaders)
  File "/home/kapil.rajput/.local/lib/python3.6/site-packages/stripe/api_requestor.py", line 151, in handle_error_response
    raise err
stripe.error.InvalidRequestError: Request req_dt5KLuhU3VOkfQ: You cannot use a Stripe token more than once: tok_1FSjfjL377PMGwZ93teT789g.
